art/attacks/hop_skip_jump.py

```diff
@@ -34,7 +36,7 @@ class HopSkipJump(Attack):
     Paper link: https://arxiv.org/abs/1904.02144
     """
     attack_params = Attack.attack_params + ['targeted', 'norm', 'max_iter', 'max_eval',
-                                            'init_eval', 'init_size', 'curr_iter']
+                                            'init_eval', 'init_size', 'curr_iter', 'batch_size']
 
     def __init__(self, classifier, targeted=False, norm=2, max_iter=50, max_eval=10000, init_eval=100, init_size=100):
```

Comment:
```suggestion
    def __init__(self, classifier, targeted=False, norm=2, max_iter=50, max_eval=10000, init_eval=100, init_size=100, batch_size=1):
```

Could you also pleas update the parameter description with 
```
:param batch_size: Batch size
        :type batch_size: `int`
```

Reply:
This attack currently does not support batching.